Cargo.toml

```diff
@@ -23,22 +23,22 @@ resolver = "2"
 [workspace.dependencies.wgc]
 package = "wgpu-core"
 git = "https://github.com/gfx-rs/wgpu"
-tag = "v25.0.2"
+tag = "v27.0.0"
```

Comment:
Now that there's some patches out, lets bump this to v27.0.2

Reply:
Just noticed that it uses git dependencies, should I change it to use `wgpu` from crates.io instead?